classes/Course/class.hubCourse.php

```diff
@@ -203,8 +203,8 @@ protected function updateAdditionalFields() {
 
 
 	protected function deleteCourse() {
+        $hist = $this->getHistoryObject();
```

Comment:
We did do that since courses that not have been delivered anymore (deleted) where always marked as freshly deleted in the log (we do not do anything with deleted courses at all, only with deleted users). I thought since I only mark them as deleted in the sync object, this would not impact other objects. You are right however, that this might change functionality if you decide to act upon those items later by changing a flag. In the new case, they would be marked as already deleted, which might not be what one expects. 

If you do not feel secure about this, drop it. It's only "logging-sugar" for us and has not important functionality. It really only sets the flag right in this case.